Code/MinimalLib/CMakeLists.txt

```diff
@@ -2,6 +2,21 @@ include_directories(${RDKit_ExternalDir})
 include_directories(${RDKit_ExternalDir}/rapidjson-1.1.0/include)
 
 if(RDK_BUILD_MINIMAL_LIB)
+    set(MINIMAL_LIB_LIBRARIES "MolInterchange_static;Abbreviations_static;"
+        "CIPLabeler_static;MolDraw2D_static;Depictor_static;RDInchiLib_static;"
+        "SubstructMatch_static;FileParsers_static;"
+        "SmilesParse_static;GraphMol_static;RDGeometryLib_static;RDGeneral_static")
+    if(RDK_BUILD_MINIMAL_LIB_RXN)
+        add_definitions(-DRDK_BUILD_MINIMAL_LIB_RXN)
```

Comment:
You should also be adding the reaction library here too, right?

Reply:
Yes, I did not notice as apparently it is automatically linked even though I did not explicitly add it.